warp10/src/main/java/io/warp10/script/formatted/DocumentationGenerator.java

```diff
@@ -92,11 +93,11 @@
     List<Object> input2 = new ArrayList<>();
     LinkedHashMap<String, String> optMap = new LinkedHashMap<>();
 
-    for (ArgumentSpecification arg: args) {
+    for (ArgumentSpecification arg: Lists.reverse(optArgs)) {
       optMap.put(arg.getName(), arg.getName() + ":" + arg.WarpScriptType());
     }
 
-    for (ArgumentSpecification arg: optArgs) {
+    for (ArgumentSpecification arg: Lists.reverse(args)) {
```

Comment:
Does the list need to be re-reversed?

Reply:
doesn't re-reversed since it return a view